widgets/src/marshaller/HipieDDL.js

```diff
@@ -60,6 +60,17 @@
                 if (val === undefined) {
                     val = item[rhsKey.toLowerCase()];
                 }
+                //  Symposium AVE Hack 
+                if (val === undefined && rhsKey.indexOf("_AVE") === rhsKey.length - 4 && item.base_count !== undefined) {
+                    var rhsSum = rhsKey.substring(0, rhsKey.length - 4) + "_SUM";
+                    val = item[rhsSum];
+                    if (val === undefined) {
+                        val = item[rhsSum.toLowerCase()];
+                    }
+                    if (val) {
+                        val /= item.base_count;
+                    }
+                }
```

Comment:
I don't think so.  

This is simply taking the the SUM and dividing by the base_count on a per row basis.

In the example I looked at the aggregate (SUM) had already been burned into the WU Result?